src/main/java/com/hyperwallet/clientsdk/model/HyperwalletUser.java

```diff
@@ -23,8 +23,11 @@
 
     public static enum Status {PRE_ACTIVATED, ACTIVATED, LOCKED, FROZEN, DE_ACTIVATED}
 
+    public static enum VerificationStatus {EXPIRED, FAILED, UNDER_REVIEW, VERIFIED, REQUIRED, NOT_REQUIRED}
```

Comment:
The verification status in Portal Docs is not updated or this code exposes more of it? 
https://portal.hyperwallet.com/docs/api/v3/resources/users/retrieve

Please sync as needed

Reply:
Ash would be creating a separate ticket to get the document updated.

Reply:
👍 cool